go/vt/vtgate/vindexes/vschema.go

```diff
@@ -60,7 +60,7 @@ const (
 // used for building routing plans.
 type VSchema struct {
 	RoutingRules      map[string]*RoutingRule `json:"routing_rules"`
-	uniqueTables      map[string]*Table
+	globalTables      map[string]*Table
```

Comment:
Renamed because although reference tables and their sources aren't unique, they can be globally routed with this PR.